phoenix-core/src/it/java/org/apache/phoenix/monitoring/BasePhoenixMetricsIT.java

```diff
@@ -110,7 +110,7 @@ void assertMutationMetrics(String tableName, int numRows, Map<String, Map<Metric
             String t = entry.getKey();
             assertEquals("Table name didn't match for mutation metrics", tableName, t);
             Map<MetricType, Long> p = entry.getValue();
-            assertEquals("There should have been four metrics", 4, p.size());
+            assertEquals("There should have been four metrics", 5, p.size());
```

Comment:
nit: five in the text or can use the number in place of text by assigning it to a variable.